src/js/config/Config.template.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
     },
     introduction: {
       enabled: false
+    },
+    'remove-oauth': {
+      enabled: false
```

Comment:
This doesn't belong here as there's no `remove-oauth` plugin in this repository.